src/pupil_labs/neon_recording/stream/gaze_stream.py

```diff
@@ -37,13 +38,7 @@ def keys(self):
 class GazeArray(Array[GazeRecord], GazeProps):
     record_class = GazeRecord
 
-
-class GazeStream(Stream[GazeArray, GazeRecord], GazeProps):
-    """Gaze data"""
-
-    data: GazeArray
-
-    def __init__(self, recording: "NeonRecording"):
+    def __new__(cls, recording: "NeonRecording"):
```

Comment:
This change might break being able to call `GazeArray('/recs/neon/gaze ps1.raw')`
```python
In: GazeArray('/recs/neon/gaze ps1.raw')
Out: 
GazeArray([(745.54865, 901.6488 ), (753.90814, 894.15027),
           (757.0492 , 889.1996 ), ..., (827.88885, 807.9901 ),
           (826.67444, 806.21234), (826.869  , 802.00385)],
          dtype=[('x', '<f4'), ('y', '<f4')])
```
We should make a test for that